javascript/chronometer.js

```diff
@@ -31,10 +31,10 @@ class Chronometer {
     return seconds;
   }
 
-getMilSeconds(){
-  let milSeconds = this.currentime % 100;
-  return milSeconds;
-}
+  getMilSeconds() {
+    let milSeconds = (this.currentime * 1000) % 1000;
+    return milSeconds;
+  }
```

Comment:
With miliseconds it is a bit more complicated. Instead of converting it here from seconds, you might want to update your `start()` method. Either add another setInterval(), or update the existing setInterval() but increase the speed 

So for example every 10miliseconds you want to do something, be increasing the milisecond counter.

Reply:
thanks for reviewing and for the hints.. :)